src/features/rewards/walletWrapper/style.ts

```diff
@@ -78,7 +78,6 @@ export const StyledContent = styled<StyledProps, 'div'>('div')`
   position: relative;
   background: #f9fbfc;
   flex: 1;
-  height: 381px;
```

Comment:
I think we need min-height based on our talk so that panel min height is fully populated summary view

Reply:
@NejcZdovc min-height added.

Now Rewards summary will at minimum be the height that it is when it has 5 line items

<img width="480" alt="screen shot 2019-02-13 at 7 48 33 pm" src="https://user-images.githubusercontent.com/8732757/52759386-e7c62d00-2fc8-11e9-88da-679c4e5f215b.png">

<img width="467" alt="screen shot 2019-02-13 at 7 49 18 pm" src="https://user-images.githubusercontent.com/8732757/52759388-e98ff080-2fc8-11e9-96f1-4972f1519c25.png">